### ISSUE TITLE
[Backend Bug Report] Login loops ( Login Delay not working)

### ISSUE THREAD

**Issue by jepke** [RealDeviceMap/RealDeviceMap#145]
**Describe the bug**
Massive quest instance aprox 250KM in diameter. Login delay op 900 just keeps looping login logout. Different accounts every new login.

When login delay set to 7200 it does not loop but puts 1 phone on delay in my case 45 minutes at most.

Right after it logged in this log entry shows up.
```
Over Logout Delay. Switching Account from XXX1 to XXX2
```
This log entry shows up as well but does not occur every loop
```
Ingoring over Logout Delay, because no account is specified.
````

**To Reproduce**
Probably to create a big instance with 2 stops far far appart.

**Expected behavior**
Pick a account scan the quest logout pick a new account scan the quest as what Login delay is suppose to do.

**Comment by Fabio1988** [comment]
Tested on latest? Still a thing?

**Comment by 123FLO321** [comment]
Fixed in 1.18.0
